<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logado</class>
 <widget class="QMainWindow" name="Logado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>369</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>347</width>
    <height>369</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>347</width>
    <height>369</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Logado</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 130, 195);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_nome">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>Olá, {nome}!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_logado">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>Você está logado!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="btn_logout">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #edd400;
border-radius: 15px;
}
QPushButton:hover{
background-color: #d1bc08;
}

</string>
      </property>
      <property name="text">
       <string>SAIR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
